tests/resources/Harbor-Pages/Vulnerability.robot

```diff
@@ -13,12 +13,15 @@ Disable Scan Schedule
      Retry Element Click  //cron-selection//button[contains(.,'SAVE')]
 
 Go To Vulnerability Config
-    Retry Element Click  //config//button[contains(.,'Vulnerability')]
+    Retry Element Click  //vulnerability-page//button[contains(.,'Vulnerability')]
```

Comment:
Do not use hard code for xpath locator, please use variable, and variable should be located in element file.